tokio-test/src/io.rs

```diff
@@ -127,6 +129,12 @@ impl Builder {
         self
     }
 
+    /// Set name of the mock IO object to include in panic messages and debug output
+    pub fn name(&mut self, name: String) -> &mut Self {
```

Comment:
How about accepting `Into<String>` for more flexibility?

For example: https://github.com/tokio-rs/tokio/blob/8507e28f89916662d2f61af823993483169d912c/tokio/src/runtime/builder.rs#L488

Reply:
Adjusted.